library/src/main/java/com/basgeekball/awesomevalidation/validators/Validator.java

```diff
@@ -155,6 +178,21 @@ private boolean checkConfirmationTypeField(ValidationHolder validationHolder, Va
         return true;
     }
 
+    private boolean checkCustomTypeField(ValidationHolder validationHolder, ValidationCallback callback) {
+        boolean valid;
+        try{
+            valid =  validationHolder.getCustomValidation().compare(validationHolder.getText());
+        }catch (Exception e){
+            valid = false;
+        }
+        if (!valid) {
+            Matcher matcher = Pattern.compile("±*").matcher(validationHolder.getText());
+            executeCallback(callback, validationHolder, matcher);
+            return false;
+        }
+        return true;
+    }
```

Comment:
There should not be any exception for this case.  And replacing the whole method body with below should be enough:

```java
boolean valid =  validationHolder.getCustomValidation().compare(validationHolder.getText());
if (!valid) {
    executeCallback(callback, validationHolder, null);
    return false;
}
return true;
```